src/DisplayBoard.cpp

```diff
@@ -0,0 +1,110 @@
+#include "DisplayBoard.hpp"
+
+namespace
+{
+    using u_int = unsigned int;
+
+    const char HORIZONTAL = '-';
+    const char VERTICAL = '|';
+    const char INTERSECTION = '+';
+    const char PADDING = ' ';
+
+
+    std::string drawPadding(const u_int width, const char sign = PADDING);
+    std::string drawEndLine(const u_int width);
+    std::string drawEmptyLine(const u_int width, const char sign = PADDING);
+    std::string drawColumns(const u_int maxElementsInRows, const u_int colNumber, const u_int heightCol);
+    std::string drawRow(const u_int maxElementsInRows);
+    std::stringstream drawBoard(const u_int width,
+                                const u_int height,
+                                const u_int widthRows,
+                                const u_int heightCol);
+}
+
+namespace DisplayBoard
+{
+    std::string displayInterface(const Board& b){
+        auto width = b.getRowsNumber();
+        auto height = b.getColsNumber();
+        auto widthRows = b.findLongestVectorInRows();
+        auto heightCol = b.findLongestVectorInCols();
+
+        auto tmp = drawBoard(width, height, widthRows, heightCol);
+        auto tmpStr = tmp.str();
+        std::cout << tmpStr << "\n";
+
+        return tmpStr;
+    }
+}
+
+namespace
+{
+    std::stringstream drawBoard(const u_int width,
+                                const u_int height,
+                                const u_int widthRows,
+                                const u_int heightCol)
```

Comment:
Why drawBoard has to have these 4 paramteres? It should have taken it directly from Board object, which may be passed here as const reference.

Reply:
Imo, this is helper method and it should not be dependent in any way from Object. It should only take params as was described in specyfication of 2.2.
@LordLukin what do you think?

Reply:
I replied above:

> Think about usability. In this form you would pass parameters like
> `drawBoard(board.getWidth(), board.getHeight(), ...)`
> So it will be easier to pass whole Board as a const &, because then you will only have to write
> `drawBoard(board);`

Reply:
done